### ISSUE TITLE
Not booting up when pressing start

### ISSUE THREAD

**Issue by FabulousAliya** [d0k3/GodMode9#612]
So I am following this guide: https://3ds.hacks.guide/finalizing-setup
When it tells me to boot pressing start while it turns on, my 3ds turns on but stays black for a long time until I turn it off to try again. Should I wait longer? I have the .firm in the payloads luma folder, and everything else has gone well. Did I set something up wrong?

**Comment by franklin608** [comment]
I was having this exact problem today too. I found this page: https://github.com/d0k3/GodMode9/issues/466

Somebody on there suggested the following:
"I have noticed that if you enter the Luma3DS settings/config screen (by holding SELECT on boot), then tapping START (to save preferences), and then pressing and holding START (or whatever button you have assigned the GM9 firm), GodMode9 boots normally, with both screens working, every time"

That worked for me. After I tapped START to save preferences, I immediately pressed and held it again and then godmode9 started up with the options that the 3ds hacks guide describes.
